lib/_cookieUtils.ts

```diff
@@ -20,24 +20,28 @@ const setCookie = (cname: string, cvalue: string, exdays: number) => {
  * @param  {[type]} cname [description]
  * @return {[type]}       [description]
  */
-const getCookie = (cname: string):string => {
-    var name = cname + "=";
-    var decodedCookie = decodeURIComponent(document.cookie);
-    var ca = decodedCookie.split(';');
-    for(var i = 0; i <ca.length; i++) {
-        var c = ca[i];
-        while (c.charAt(0) == ' ') {
-            c = c.substring(1);
-        }
-        if (c.indexOf(name) == 0) {
-            return c.substring(name.length, c.length);
-        }
+export const getCookie = (cname: string): string => {
+  const name = cname + "=";
+  const ca = document.cookie.split(";");
+  for (let i = 0; i < ca.length; i++) {
+    let c = ca[i];
+    while (c.charAt(0) === " ") {
+      c = c.substring(1);
     }
-    return "";
+    if (c.indexOf(name) === 0) {
+      try {
+        return decodeURIComponent(c.substring(name.length, c.length));
```

Comment:
decoding isn't necessary here because the only possible values for the uuid are within ascii

Reply:
good point! fixed heree 44bad6